components/list/index.zh-CN.md

```
@@ -53,7 +53,7 @@ List 组件已经进入废弃阶段，将于下个 major 版本移除，请改
 | footer | 列表底部 | ReactNode | - |  | × |
 | grid | 列表栅格配置 | [object](#list-grid-props) | - |  | × |
 | header | 列表头部 | ReactNode | - |  | × |
-| itemLayout | 设置 `List.Item` 布局，设置成 `vertical` 则竖直样式显示，默认横排 | string | - |  | × |
+| itemLayout | 设置 `List.Item` 布局，设置成 `vertical` 则竖直样式显示，默认横排 | `horizontal` | `vertical` | `horizontal` |  | × |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 6; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)